packages/commonwealth/client/scripts/views/pages/search/index.tsx

```diff
@@ -293,54 +300,62 @@ const SearchPage = () => {
             </CWTabsRow>
           </div>
           <>
-            {isLoading && <PageLoading />}
-            {!isLoading && (
+            {sharedQueryOptions?.searchTerm?.length === 0 && (
+              <ErrorPage message="No search term provided!" />
+            )}
+            {sharedQueryOptions?.searchTerm?.length > 0 && (
               <>
-                <CWText className="search-results-caption">
-                  {totalResultsText} matching &apos;{queryParams.q}&apos;{' '}
-                  {scopeText}
-                  {community !== 'all_communities' && !app.isCustomDomain() && (
-                    <a
-                      href="#"
-                      className="search-all-communities"
-                      onClick={handleSearchAllCommunities}
-                    >
-                      Search all communities?
-                    </a>
-                  )}
-                </CWText>
-                {VISIBLE_TABS.length > 0 &&
-                  [SearchScope.Threads, SearchScope.Replies].includes(
-                    activeTab,
-                  ) && (
-                    <div className="search-results-filters">
-                      <CWText type="h5">Sort By:</CWText>
-                      <CWDropdown
-                        label=""
-                        onSelect={handleSortChange}
-                        initialValue={{
-                          // @ts-expect-error <StrictNullChecks/>
-                          label: queryParams.sort,
-                          // @ts-expect-error <StrictNullChecks/>
-                          value: queryParams.sort,
-                        }}
-                        options={Object.keys(SearchSort).map((k) => ({
-                          label: k,
-                          value: k,
-                        }))}
-                      />
+                {isLoading && <PageLoading />}
+                {!isLoading && (
```

Comment:
I think using ternary here would be better

{ isLoading ? <PageLoading/> : content }